sql/core/src/main/scala/org/apache/spark/sql/execution/Sort.scala

```diff
@@ -87,10 +89,12 @@ case class Sort(
       // Remember spill data size of this task before execute this operator so that we can
       // figure out how many bytes we spilled for this operator.
       val spillSizeBefore = metrics.memoryBytesSpilled
+      val beforeSort = System.currentTimeMillis()
```

Comment:
Should we use nanoTime() instead of currentTimeMillis(), which is not guaranteed to be monotonic?